src/compute/hash.rs

```diff
@@ -29,7 +30,8 @@ pub fn hash_primitive<T: NativeType + Hash>(array: &PrimitiveArray<T>) -> Primit
 }
 
 #[multiversion]
-#[clone(target = "x86_64+aes+sse3+ssse3+avx+avx2")]
+#[clone(target = "[x86|x86_64]+avx+avx2")]
+#[clone(target = "x86+sse")]
```

Comment:
I'm not sure if it makes sense, x86 is pretty old by now, so not sure it makes sense to add it here if we don't know the performance impact.